js/plugin/popups.ts

```diff
@@ -20,8 +22,10 @@ class BrowserModPopup extends LitElement {
   @property() right_button;
   @property() left_button;
   @property() dismissable;
+  @property({ reflect: true }) timeout_hide_progress;
   @property({ reflect: true }) wide;
   @property({ reflect: true }) fullscreen;
+  @property({ reflect: true }) classic;
```

Comment:
This should be on the todo-list to fix as well. Those three properties are pretty miuch mutually exclusive, and the order of precedence is unclear.

Reply:
In a way these are standardising different users wishes for popup styling. Perhaps taking a theme approach might make sense. The current `size` views (`style-tag`?) can be then taken from a theme/style if available, or current styles used as default. This would allow for users to extend styling with their own. e.g. `style: my-own-popup-style`

Reply:
Created issue #842